gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/services/FileImporter.java

```diff
@@ -25,26 +27,40 @@
 public class FileImporter {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(FileImporter.class);
+    public static final String CRAC_CIM_CRAC_CREATION_PARAMETERS_PT_ES_JSON = "/crac/CimCracCreationParameters_PT-ES.json";
+    public static final String CRAC_CIM_CRAC_CREATION_PARAMETERS_FR_ES_JSON = "/crac/CimCracCreationParameters_FR-ES.json";
     private final UrlValidationService urlValidationService;
 
     public FileImporter(UrlValidationService urlValidationService) {
         this.urlValidationService = urlValidationService;
     }
 
-    public CimCrac importCimCrac(String cracUrl) {
+    public CimCrac importCimCrac(SweRequest sweRequest) {
         LOGGER.info("Importing Cim Crac file from url");
-        InputStream cracInputStream = urlValidationService.openUrlStream(cracUrl);
+        InputStream cracInputStream = urlValidationService.openUrlStream(sweRequest.getCrac().getUrl());
         CimCracImporter cimCracImporter = new CimCracImporter();
         return cimCracImporter.importNativeCrac(cracInputStream);
     }
 
-    public Crac importCrac(CimCrac cimCrac, OffsetDateTime targetProcessDateTime, Network network) {
+    public Crac importCrac(CimCrac cimCrac, OffsetDateTime targetProcessDateTime, Network network, CracCreationParameters params) {
         LOGGER.info("Importing native Crac from Cim Crac and Network for process date: {}", targetProcessDateTime);
-        return CracCreators.createCrac(cimCrac, network, targetProcessDateTime).getCrac();
+        return CracCreators.createCrac(cimCrac, network, targetProcessDateTime, params).getCrac();
     }
 
-    public Crac importCimCracFromUrlWithNetwork(SweRequest sweRequest, Network network) {
-        return importCrac(importCimCrac(sweRequest.getCrac().getUrl()), sweRequest.getTargetProcessDateTime(), network);
+    public Crac importCracFromUrlWithNetworkFrEs(CimCrac cimCrac, SweRequest sweRequest, Network network) {
+        return importCrac(
```

Comment:
Do you think its better to have only one function with the border as a parameter instead of those 2 functions almost the same ?